services/metadata/src/metadata/services/templates.clj

```diff
@@ -14,6 +17,30 @@
       (ca/assert-found "metadata template" template-id)
       (remove-nil-values)))
 
+(defn view-template-csv
+  [template-id]
+  (csv/csv-string [
+    (->> (view-template template-id)
+      :attributes
+      (map :name)
+      (cons "file name or path"))]))
+
+(defn view-template-guide
+  [template-id]
+  (csv/csv-string
+    (->> (view-template template-id)
+      :attributes
+      (map (juxt :name :description :required :type #(if (:values %) (string/join ", " (map :value (:values %))) "")))
+      (cons ["attribute name", "attribute description", "required (If you cannot provide,  enter 'not collected', 'not applicable' or 'missing'.)", "value type definition", "enum value options (you must enter one of these values)"]))))
+
+(defn csv-download-resp
```

Comment:
Minor suggestion: I wonder if this function should be defined up at the `metadata.routes.templates` level. Then the `ok` function and the details of the HTTP response could be kept there?